aws/utils/logging.cc

```diff
@@ -147,6 +147,11 @@ class AwsBoostLogInterface : public Aws::Utils::Logging::LogSystemInterface {
     va_end(args);
   }
 
+  // cpp sdk made this virtual so needed to have a dummy implementation for it for build to pass
```

Comment:
What does it mean for a method to be virtual? And what is `vaLog` supposed to do in the first place?

Reply:
Virtual allows derived classes to override the method, meaning they can provide their own implementation of the method. Based on this the [PR](https://github.com/aws/aws-sdk-cpp/blob/main/src/aws-cpp-sdk-core/include/aws/core/utils/logging/LogSystemInterface.h#L39-L42) i think `vaLog` is just another variety of `log` that can take more arguments. We are not using this method in our code so this should be fine to ignore.